bootstraptest/test_yjit.rb

```diff
@@ -1832,6 +1832,18 @@ def jittable_method
   jittable_method
 }
 
+# test getbyte on string class
+assert_equal '[97, nil]', %q{
+  def getbyte(s, i)
+    s.getbyte(i) || :nil
+  end
+
+  getbyte("a", 0)
+  getbyte("a", 0)
+
+  [getbyte("a", 0), getbyte("a", 1)]
```

Comment:
Could you also test with indices -1 and -2 just for completeness? I know it will work, but it's good to test things thoroughly in case of future refactorings.

Reply:
Thank you! I updated as part of [`38590fe`](https://github.com/ruby/ruby/compare/76251be9e26e088d319600d5c9b895d609cf89ac..38590fe1b9e0bcb3a226390714f9bf4f19011eae)